### PR TITLE
New CPE210/... v3 versions strings

### PR DESCRIPTION
**Pull request by NeoRaider**
Clean up aliases (moving minor version strings to manifest_aliases) and add the new CPE210 3.1 and 3.20 strings to allow upgrades.

Closes: #2080

### CODE REVIEW

File: targets/ar71xx-generic [hunk 1]
```diff
@@ -251,18 +251,50 @@ if (env.GLUON_REGION or '') ~= '' then
 	tplink_region_suffix = '-' .. env.GLUON_REGION
 end
 
-device('tp-link-cpe210-v1.0', 'cpe210-220-v1', {
-	aliases = {'tp-link-cpe210-v1.1', 'tp-link-cpe220-v1.1'},
+device('tp-link-cpe210-v1', 'cpe210-220-v1', {
+	aliases = {
+		'tp-link-cpe220-v1',
```
Comment (rotanid) [inline targets/ar71xx-generic:256]: either i don't understand it or here is an issue, please check again

File: targets/ar71xx-generic [hunk 2]
```diff
@@ -251,18 +251,50 @@ if (env.GLUON_REGION or '') ~= '' then
...
+	},
 })
-device('tp-link-cpe210-v2.0', 'cpe210-v2')
-device('tp-link-cpe210-v3.0', 'cpe210-v3')
 
-device('tp-link-cpe510-v1.0', 'cpe510-520-v1', {
-	aliases = {'tp-link-cpe510-v1.1', 'tp-link-cpe520-v1.1'},
+device('tp-link-cpe510-v1', 'cpe510-520-v1', {
+	aliases = {
+		'tp-link-cpe520-v1',
```
Comment (rotanid) [inline targets/ar71xx-generic:279]: either i don't understand it or here is an issue, please check again

File: targets/ar71xx-generic (hunk 1 above)
Comment (NeoRaider) [reply targets/ar71xx-generic:256]: The device will be listed as CPE210 v1 and CPE220 v1 in the sysupgrade / factory output directories (with the latter being a symlink / alias). These are the names that users will look for when searching for images for their devices.

**Comment by rotanid** [comment]
merged, backports to stable branches necessary according to @NeoRaider

**Comment by rotanid** [comment]
backported via cherry-pick to v2020.1.x and v2020.2.x

**Pull request merged**